src/database/sqlite/sqlite_aclk_alert.c

```diff
@@ -219,245 +213,438 @@ static inline char *sqlite3_text_strdupz_empty(sqlite3_stmt *res, int iCol) {
     return strdupz(ret);
 }
 
+#define SQL_UPDATE_ALERT_VERSION                                                                                       \
+    "INSERT INTO alert_version (health_log_id, unique_id, status, version, date_submitted)"                            \
+    " VALUES (@health_log_id, @unique_id, @status, @version, @date_submitted)"                                         \
+    " ON CONFLICT(health_log_id) DO UPDATE SET status = excluded.status, version = excluded.version, "                 \
+    " unique_id=excluded.unique_id, date_submitted=excluded.date_submitted"
+
+//
+// Store a new alert transition along with the version after sending to the cloud
+//   - Update an existing alert with the updated version, status, transition and date submitted
+//
+static void sql_update_alert_version(int64_t health_log_id, int64_t unique_id, RRDCALC_STATUS status, uint64_t version)
+{
+    static __thread sqlite3_stmt *res = NULL;
+
+    if (!PREPARE_COMPILED_STATEMENT(db_meta, SQL_UPDATE_ALERT_VERSION, &res))
+        return;
+
+    int param = 0;
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, health_log_id));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, unique_id));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int(res, ++param, status));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, version));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, now_realtime_sec()));
+
+    param = 0;
+    int rc = sqlite3_step_monitored(res);
+    if (rc != SQLITE_DONE)
+        error_report("Failed to execute sql_update_alert_version");
 
-static void aclk_push_alert_event(struct aclk_sync_cfg_t *wc __maybe_unused)
+done:
+    REPORT_BIND_FAIL(res, param);
+    SQLITE_RESET(res);
+}
+
+#define SQL_SELECT_ALERT_TO_DUMMY                                                                                      \
+    "SELECT aa.sequence_id, hld.unique_id, hld.when_key, hld.new_status, hld.health_log_id"                            \
+    " FROM health_log hl, aclk_queue aa, alert_hash ha, health_log_detail hld"                                         \
```

Comment:
It looks like our tables have different patterns:

1st pattern (abbreviation):
- health_log: hl
- helath_log_detail: hld

2nd pattern (inverse abbreviation):
- alert_hash: ha

3rd pattern ( I am still thinking about): 
- aclk_queue: aa

if there is no special reason for this, please, can we use only one in this query? :) Thanks!

Reply:
yes, leftovers .. I will adjust as needed